.eslintrc.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
   extends: [
     'eslint:recommended',
     'plugin:ember/recommended',
-    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/eslint-recommended'
```

Comment:
don't know where you get `    'plugin:@typescript-eslint/eslint-recommended'` from, I couldn't find any references to it. only to `plugin:@typescript-eslint/recommended`